SPECS/openjdk/openjdk17.spec

```diff
@@ -107,6 +108,18 @@ sh ./configur* \
     --with-freetype-lib=%{_libdir} \
     --with-stdc++lib=dynamic \
     --disable-warnings-as-errors
+%else
```

Comment:
Probably this will suffice for both aarch64 and x86_64 without code duplication and condition checking.

```
sh ./configur* \
    --with-target-bits=64 \
    --enable-headless-only \
    --with-extra-cxxflags="-Wno-error -fno-delete-null-pointer-checks -fno-lifetime-dse" \
    --with-extra-cflags="-fno-delete-null-pointer-checks -Wno-error -fno-lifetime-dse" \
    --with-freetype-include=%{_includedir}/freetype2 \
    --with-freetype-lib=%{_libdir} \
    --with-stdc++lib=dynamic \
    --build=%{_build} \
    --disable-warnings-as-errors
```

Reference: https://github.com/rpm-software-management/rpm/blob/master/macros.in#L1066

Please give this a try. Thanks for your contribution and continues engagement to make our product better.